sdv/data_processing/data_processor.py

```diff
@@ -355,25 +349,35 @@ def _fit_numerical_formatters(self, data):
                 )
                 self.formatters[column_name].learn_format(data[column_name])
 
+    def _prepare_fitting(self, data):
```

Comment:
we could make this method public honestly

Reply:
Sounds good 👍🏻